proto/one/common/notification/signalr/signalr_one_hub_event.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+import "signalr_one_hub_event_content.proto";
+import "jsonTicksDateTime.proto";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+message SignalROneHubEvent {
+    string area = 1;
+    string name = 2;
+    JsonTicksDateTime time = 3;
```

Comment:
I think this could be more descriptive.  HistorianData uses the name dateTimeUTC for a similar field.